src/test/resources/automationpractice.xml

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Test Suite for testing http://automationpractice.com/ website" verbose="1">
+    <listeners>
+        <listener class-name="com.utils.TestListener"/>
+    </listeners>
+    <test name="Main Page Tests">
+        <classes>
+            <class name="com.automationpracticetest.MainPageTest"/>
```

Comment:
You can use packages instead classes so not to specify each file
e.g. <packages> <package name="automationpracticetest"></package> </packages>